src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -301,6 +301,11 @@ private void DisplaySuggestions(object sender, RoutedEventArgs e)
         private void ShowLowConfidenceResults(object sender, RoutedEventArgs e)
         {
             ViewModel.ShowLowConfidenceResults();
+            //Tracking Analytics when the Low Confidence combobox (located in the Autocomplete popup)  is clicked
+            Analytics.TrackEvent(
+                    Actions.ExpandConfidenceLevel,
+                    Categories.NodeAutoCompleteOperations,
+                    "LowConfidenceResults");
```

Comment:
For example, I would re-use `Expanded` action here for simplicity
```
Analytics.TrackEvent(
                    Actions.Expanded,
                    Categories.NodeAutoCompleteOperations,
                    nameof(ShowLowConfidenceResults));
```

Reply:
The Expanded action was re-used in this line, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13581/commits/0655f6d242fa449dc828a75d8634567d3b5737f4